.gitlab/common/macos.yml

```diff
@@ -1,6 +1,38 @@
 ---
 # This is the scripts to be executed on the Gitlab macOS runners before every job.
 # We don't have virtualization now so we need to clean the environment and install the proper dependencies before every job.
+.list_go_versions_commands: &list_go_versions_commands
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why doing yaml references while we can do ⭐  gitlab references ⭐  ?

Reply:
You're right, thanks !

People are waiting for this ASAP so I'm still merging the PR now. I'm fixing this in another PR though !